tests/lib/rules/require-ember-lifeline.js

```diff
@@ -1,26 +1,31 @@
 const rule = require('../../../lib/rules/require-ember-lifeline');
-const MESSAGE = rule.meta.message;
+const getMessage = rule.meta.message;
 const RuleTester = require('eslint').RuleTester;
 const ruleTester = new RuleTester();
```

Comment:
IDK I think it's more than a nit. We should prob make this change to all the tests. Figure it's still early days within this repo and we can solidify the ideals.

Essentially we have lines of code within tests that are redundant. I'm all for a balance of succinct yet grokkable. Thoughts?